api-reference/timescaledb/tag-overview.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: API reference tag overview
+description: The TimescaleDB API reference includes tags to categorize the available SQL functions and views
+tags: [licenses, toolkit, experimental]
+products: [cloud, mst, self_hosted]
+---
+
+import { TIMESCALE_DB, TOOLKIT_LONG, TDB_COMMUNITY } from '/snippets/vars.mdx';
+
+The {TIMESCALE_DB} API Reference uses tags to categorize functions. The tags are `Community`, `Experimental`, `Toolkit`,
+and `Experimental (Toolkit)`. This section explains each tag.
+
+## Community
+
+This tag indicates that the function is available under TimescaleDB Community Edition, and are not available under the
+Apache 2 Edition. For more information, visit our [TimescaleDB License comparison sheet][tsl-comparison].
+
+## Experimental (TimescaleDB Experimental Schema)
```

Comment:
This no longer makes sense, as we now say "early access". I also think the Community badge should change to say "Community edition" because it's more self-explanatory.